### PR TITLE
March 20 updates

### PR DESCRIPTION
**Pull request by kflemin**
adding L000 BRICR use case example files
attempting to dockerize adding the TestSuite repo

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,9 +43,15 @@ WORKDIR /srv/selection-tool
 COPY /requirements.txt /srv/selection-tool/requirements.txt
 RUN pip install -r requirements.txt
 
-#install the schematron-nokogiri gem
-RUN gem install schematron-nokogiri
+#install the schematron-nokogiri gem from BuildingSync org
+WORKDIR /srv
+RUN git clone https://github.com/BuildingSync/schematron
```
Comment (nllong) [inline Dockerfile]: yay! looks like we had to do some tweaks?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ This is the repository for the BuildingSync Validator web application.
 	```bash
 	pip install -r requirements.txt
 	```
+1.  Install the schematron-nokogiri gem for schematron use case validation.  We are using a forked version to handle OpenStudio Simulation use cases which contain abstract patterns.
```
Comment (nllong) [inline README.md]: 👍

File: Dockerfile (hunk 1 above)
Comment (kflemin) [reply Dockerfile]: yep!

**Comment by nllong** [comment]
👍 thanks @kflemin .

**Pull request merged**